satellite/console/consoleweb/consoleql/query.go

```diff
@@ -90,6 +97,50 @@ func rootQuery(service *console.Service, types *TypeCreator) *graphql.Object {
 					return tokenWrapper{Token: token}, nil
 				},
 			},
+			ForgotPasswordQuery: &graphql.Field{
+				Type: graphql.Boolean,
+				Args: graphql.FieldConfigArgument{
+					FieldEmail: &graphql.ArgumentConfig{
+						Type: graphql.NewNonNull(graphql.String),
+					},
+				},
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					email, _ := p.Args[FieldEmail].(string)
+
+					user, err := service.GetUserByEmail(p.Context, email)
```

Comment:
You need to use `console.Service` or create a new service. Using mail service ends up mixing layers too much.